apps/condo/domains/marketplace/schema/MarketCategory.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Generated by `createschema marketplace.MarketCategory 'name:Text; parentCategory:Relationship:MarketCategory:CASCADE;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File, Url } = require('@keystonejs/fields')
+const get = require('lodash/get')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { Json, LocalizedText } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/marketplace/access/MarketCategory')
+const { MARKET_CATEGORY_ERRORS } = require('@condo/domains/marketplace/constants')
+
+
+const MarketCategory = new GQLListSchema('MarketCategory', {
```

Comment:
Md do that if we delete a category, all subcategories are deleted too?

Reply:
I'm not sure if cascade deletion works now, so no.

Reply:
After cascade deletion works, the following behavior is expected. When deleting a root category, all inherited categories are also deleted. If you need to delete something in the middle while keeping the branch below, then you will need to do the following. Create a category next to the one to be deleted, reassign the inherited ones to the new category and then delete the unnecessary one